# Glioblastoma_P1_AK
# Project Overview
This project employs systems biology methodologies for drug repositioning aimed at developing alternative treatments for Glioblastoma. It utilizes publicly available datasets from the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas Program (TCGA) cohorts. Our analyses incorporate several computational techniques to evaluate the efficacy and potential of repurposed drugs.

# Data Sources
CGGA Cohort
TCGA Cohort
Datasets are publicly available and were utilized for this study to ensure reproducibility and accessibility.

# Analysis Workflow

# Differential Gene Expression Analysis (DEG)
Performed using the DESeq2 package in R. This analysis helps in identifying genes that show statistically significant differences in expression between conditions.

# Survival Analysis
Cox proportional hazards models were applied to explore the relationship between gene expression and patient survival times. This analysis was carried out using R.

# Gene Set Enrichment Analysis
Conducted via R to identify classes of genes that are over-represented in a large set of genes or proteins and may have an association with disease phenotypes.

# Weighted Gene Co-expression Network Analysis (WGCNA)
This method was used to identify clusters (modules) of highly correlated genes, to relate modules to one another. WGCNA provides insights into network features of gene expression data.

# Figure Production
Output figures were generated using R, providing visual insights into the data and analysis results.

# Usage
Scripts and methodologies used for the analyses are outlined for reproducibility.

# Contribution

# License

This README provides a comprehensive guide to the methodologies and data sources employed in the study, aiming to facilitate further research and development in the field of Glioblastoma treatment.
